### DISCUSSION TITLE
Open folder in same window

### DISCUSSION THREAD

**Discussion by marvinengelmann** [zed-industries/zed#20407] [Feature Requests]
Is there a settings option to open a remote project or local folder in current window?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
